studybuddies/src/app/app.component.ts

```diff
@@ -1,10 +1,47 @@
-import { Component } from '@angular/core';
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+import { User } from './models/user';
+import { AuthService } from './services/auth.service';
+import { TokenStorageService } from './services/token-storage.service';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.css']
 })
-export class AppComponent {
+export class AppComponent implements OnInit{
   title = 'studybuddies';
+  private role: string = '';
+  isLoggedIn = false;
+  showAdminBoard = false;
+  showStudentBoard = false;
+  showTutorBoard = false;
+  username?: string;
+
+  constructor(private tokenStorageService: TokenStorageService, private router: Router,) { }
+
+  ngOnInit(): void {
+    console.log(this.isLoggedIn)
+    console.log(this.tokenStorageService.getUser())
+    if(this.tokenStorageService.getUser()) {
+      this.isLoggedIn = true
+    }
+    console.log(this.isLoggedIn)
+    if (this.isLoggedIn) {
+      const user = this.tokenStorageService.getUser();
+      this.role = user.role;
+
+      this.showAdminBoard = this.role == 'admin';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Team-Studybuddies/Studybuddies-Frontend/pullRequest?prid=7203036)
```suggestion
      this.showAdminBoard = this.role == "admin";
```